Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -201,8 +204,15 @@ void SendAIBuffer(const short* samples, unsigned int num_samples)
 
 void StartAudioDump()
 {
-  std::string audio_file_name_dtk = File::GetUserPath(D_DUMPAUDIO_IDX) + "dtkdump.wav";
-  std::string audio_file_name_dsp = File::GetUserPath(D_DUMPAUDIO_IDX) + "dspdump.wav";
+  std::time_t start_time = std::time(nullptr);
+
+  std::string path_prefix = File::GetUserPath(D_DUMPAUDIO_IDX) + SConfig::GetInstance().GetGameID();
+
+  std::string base_name =
+      fmt::format("{}_{:%Y-%m-%d_%H-%M-%S}", path_prefix, *std::localtime(&start_time));
+
+  std::string audio_file_name_dtk = fmt::format("{}_{}", base_name, "dtkdump.wav");
```

Comment:
```suggestion
  const std::string audio_file_name_dtk = fmt::format("dtkdump.wav_{}", base_name);
```

similarly for dspdump.wav below

Reply:
Fixed. Just noting that I'm doing `fmt::format("{}_dtkdump.wav", base_name)` and not `fmt::format("dtkdump.wav_{}", base_name)` as that would append the base after the file extension.